content/docs/rfcs/16/README.md

```diff
@@ -51,6 +51,7 @@ Refer to [`Waku Message` specification](https://github.com/vacp2p/specs/blob/mas
 | `payload` | `String` | mandatory | The message payload as a hex encoded data string |
 | `contentTopic` | `String` | optional | Message content topic for optional content-based filtering |
 | `version` | `Number` | optional | Message version. Used to indicate type of payload encryption. Default version is 0 (no payload encryption). |
+| `timestamp` | `Number` | optional | The time at which the message is generated by its sender. This field holds the Unix epoch time in seconds as a 64-bits float value. |
```

Comment:
As it is a float value, does it mean milliseconds are accepted? If so, it'd be good to state it. If not, then why using a float?

Reply:
Yes, it supports up to microseconds on the fractional part. Clarified, Thanks!

Reply:
The precision though seems to be system dependent. We may need to change this to `uint64` to capture a specific resolution.